dfastmi/kernel/typehints.py

```diff
@@ -0,0 +1,7 @@
+from typing import Tuple, Optional
+
+QLevels = Tuple[float, float, float, float]
```

Comment:
Perhaps try to document the values within the tuples

Reply:
Done